frontend/src/main/components/Students/StudentsTable.js

```diff
@@ -22,14 +21,12 @@ export default function StudentsTable({ students, currentUser }) {
         }
     ];
 
-    // Stryker disable ArrayDeclaration : [columns] and [students] are performance optimization; mutation preserves correctness
-    const memoizedColumns = React.useMemo(() => columns, [columns]);
-    const memoizedDates = React.useMemo(() => students, [students]);
-    // Stryker enable ArrayDeclaration
+    // Stryker disable next-line ArrayDeclaration : [students] is a performance optimization
+    const memoizedDates = useMemo(() => students, [students]);
```

Comment:
While you're cleaning up this code, you might as well remove this line as well. It's an improper usage of `useMemo` for the same reason ESLint complained about the other line.